02-Authentication-HS256/config/auth0.yml

```diff
@@ -0,0 +1,4 @@
+development:
+  domain: <%= ENV["AUTH0_DOMAIN"] %>
+  audience: <%= ENV["AUTH0_AUDIENCE"] %>
```

Comment:
In the `auth0.yml` file for RS256 we're using `AUTH0_API_AUDIENCE` instead of `AUTH0_AUDIENCE`. Let's use the same env var for the same value.